pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -195,11 +347,7 @@ func testOperatorStateTransitions(events monitorapi.Intervals, conditionTypes []
 				if len(concurrentE2E) > 0 {
 					failure = fmt.Sprintf("%s\n%d tests failed during this blip (%v to %v): %v", failure, len(concurrentE2E), eventInterval.From, eventInterval.From, strings.Join(concurrentE2E, "\n"))
 				}
-				var Config *rest.Config
-				if len(clientConfig) > 0 {
-					Config = clientConfig[0]
-				}
-				exception, err := except(operatorName, condition, Config)
+				exception, err := except(operatorName, condition, eventInterval, clientConfig)
```

Comment:
We'll need to bubble up the SNO signal from the except or determine it outside of the except function, so we can decide when to append the `fatal` array function. [Something like this would also be preferable](https://github.com/openshift/origin/pull/28872/files#diff-7f3e3f1bd5edd0c5d4b9963c7074fb65d4145dc3ad3ba42190d3bf713f98addcR222-R224)

Reply:
@eggfoobar doe [this](https://github.com/openshift/origin/blob/2c47e0a568f7dbe15763d3d952e259f27b5d08b8/pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go#L57) do what you need?  Looks like it is returning an exception for single node.

Reply:
Uhh, nope I think the logic might be reversed.  That is an exception for `testStableSystemOperatorStateTransitions` which we don't want to allow correct?  We want the exception added for the upgrade case only handled in [testUpgradeOperatorStateTransitions](https://github.com/openshift/origin/blob/2c47e0a568f7dbe15763d3d952e259f27b5d08b8/pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go#L165C6-L165C41)

Reply:
Yup, exactly, we'll need to do the same for the upgrade func [here](https://github.com/openshift/origin/pull/28851/files#diff-7f3e3f1bd5edd0c5d4b9963c7074fb65d4145dc3ad3ba42190d3bf713f98addcR197)

Reply:
You're right, we probably don't need the stable check